test-codecov-package/test-codecov-package.txt

```diff
@@ -4,7 +4,7 @@
 // RUN: rm -rf %t.dir
 // RUN: mkdir -p %t.dir
 // RUN: cp -r %S/foo %t.dir/
-// RUN: %{swift-test} --package-path %t.dir/foo --enable-code-coverage -v 2>&1 | tee %t.build-log
+// RUN: %{swift} test --package-path %t.dir/foo --enable-code-coverage -v 2>&1 | tee %t.build-log
```

Comment:
why this change?

Reply:
This line is a revert from the previous PR: this specific test can't work correctly against a custom SwiftPM binary because it expects to find code coverage information in the same folder. It basically only works with a toolchain.

Reply:
Cool, feel free to test and merge this!